### ISSUE TITLE
ability to do module-install ps-spotify

### ISSUE THREAD

**Issue by vmiheer** [bmsimons/ps-spotify#4]

**Comment by dimapasko** [comment]
Yeah, it would be really nice to have this module at https://www.powershellgallery.com/

**Comment by Mark-Foresta** [comment]
I publish to a PowerShell get Repo internally for work if you need a hand?

**Comment by MeanDoubleStain** [comment]
I am stranded at getting the refresh token. I can't seem to find the right .NET SDK for the dotnet-core-spotify-authentication, it keeps coming back with "dotnet-core-spotify-authentication.csproj : error NU1202: Package Microsoft.AspNetCore.All 2.1.5 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0). Package Microsoft.AspNetCore.All 2.1.5 supports: netcoreapp2.1 (.NETCoreApp,Version=v2.1)" So far I've tried 2.2, 2.0 preview and 2.1, with the "dotnet restore" everytime. Is there a more clearcut way of getting the refresh token?

EDIT: I have updated the "dotnet-core-spotify-authentication.csproj" part <TargetFramework>netcoreapp2.0</TargetFramework> to <TargetFramework>netcoreapp2.1</TargetFramework> and now the app runs. I have never touched any of this, but am used to land on planet F*ckyou, on with the show!
